pyfr/solvers/baseadvec/inters.py

```diff
@@ -94,3 +98,33 @@ def _eval_opts(self, opts, default=None):
             return [npeval(cfg.get(sect, k, default), cc) for k in opts]
         else:
             return [npeval(cfg.get(sect, k), cc) for k in opts]
+
+    def _exp_opts(self, opts, lhs, default=None):
```

Comment:
Okay.  If you can do some local testing to ensure that everything works as expected when default is not None then I'm fine with it.  (I am always cautious about comitting code that might be untested.)